pkgs/by-name/mu/multiqc/package.nix

```
@@ -0,0 +1,117 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  procps,
+  testers,
+  multiqc,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "multiqc";
+  version = "1.26";
+
+  disabled = python3Packages.pythonOlder "3.8";
+
+  # Two data sources. One for the code, another for the test data
+  srcs = [
+    (fetchFromGitHub {
+      name = pname;
+      owner = "MultiQC";
+      repo = "MultiQC";
+      rev = "refs/tags/v${version}";
+      hash = "sha256-MPAw6gG/3LzdskkDXOTDEM1NpG0sH9GvklYFQ1ZXWIs=";
+    })
+    (fetchFromGitHub {
+      owner = "MultiQC";
+      repo = "test-data";
+      rev = "67435083a8bfa228dca3dda7d835facef15fc2c7";
+      hash = "sha256-oYmPIJSy6dOKPcMr3B4foGoWcerA29x0XeGoU4dSYsA=";
+      name = "test-data";
+    })
+  ];
+
+  sourceRoot = pname;
+
+  postPatch = ''
+    substituteInPlace multiqc/plots/plotly/plot.py \
+      --replace-fail '"'ps'"' '"'${procps}/bin/ps'"'
```

```suggestion
      --replace-fail '"'ps'"' '"'${lib.getExe' procps "ps"}'"'
```
Please use the `lib.getExe` pattern instead.

Still, why is this failing without an absolute path?

---

Upstream uses `subprocess.run()` with `ps aux` , and the executable. I've added a comment.